packages/story-editor/src/components/library/panes/text/fontPreview.js

```diff
@@ -79,7 +89,7 @@ const DragContainer = styled.div`
   line-height: ${({ lineHeight }) => lineHeight}px;
 `;
 
-function FontPreview({ title, element, insertPreset, getPosition }) {
+function FontPreview({ title, element, insertPreset, getPosition, index }) {
```

Comment:
It might start needing memo in #10609 -- I've added a note already into a todo list there to check Performance to ensure it will only re-renders when actually needed. Let's revisit there.